osu.Game.Rulesets.Osu/Edit/Blueprints/Sliders/SliderSelectionBlueprint.cs

```diff
@@ -100,6 +105,8 @@ public override bool HandleQuickDeletion()
             return true;
         }
 
+        private bool hasSingleObjectSelected => editorBeatmap == null || selectedObjects.Count == 1;
```

Comment:
How did this change come to be? `hasSingleObjectSelected` being `true` if `editorBeatmap` is null blows my mind a bit. If the purpose of this property has changed somehow, can we at least rename it to read less WTF?

Reply:
It's a bit of an edge case to make tests work.. I just added it to guarantee existing behaviour didn't change, but may be worth removing and fixing any tests themselves to match (if any actually break).

Reply:
Sounds like there's a nonzero chance that addressing the other review comment I left may render this change unnecessary. Probably worth a go.

Reply:
Seems that's the case, I've removed the weird null check here now.